### DISCUSSION TITLE
Getting rid of create-only migration

### DISCUSSION THREAD

**Discussion by Oscar-21** [prisma/prisma#15912] [Q&A]
```bash
$ npx prisma migrate dev --name test-mig --create-only
```
If I run a create only migration and don't want to run it, should I delete the file manually? Is it safe to keep and ignore and new migrations won't be affected?

**Comment by nurul3101** [comment]
Hello @Oscar-21 👋 

If you don't want to run a migration then you would need to remove the file, if the migration file which is created with `--create-only` flag isn't removed then the next time `npx prisma migrate dev` is executed the migration file would be applied.
Let me know if you have any other queries.

**Reply by vladi-stevanovic** [reply]
:wave: Hello @Oscar-21 , I’m going through pending GitHub Discussions to check if you need any help. Is this issue still open?

Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! :raised_hands:

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
